src/pages/en/upgrade-astro.mdx

```diff
@@ -0,0 +1,280 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Upgrade Guide
+description: How to upgrade your project to the latest version of Astro.
+i18nReady: false
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+
+This guide will help you migrate from Astro v1 to Astro v2.
+
+Need to upgrade an older project to v1? See our [older migration guide](https://github.com/withastro/docs/blob/aa7e4f568f418657eba297778729a8aee2a36409/src/pages/en/migrate.mdx).
+
+You can update your project's version of Astro to the latest version using your package manager. If you're using Astro integrations, please also update those to the latest version.
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  # updates the astro dependency:
+  npm upgrade astro
+  # or, to update all dependencies:
+  npm upgrade
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  # updates the astro dependency:
+  pnpm upgrade astro
+  # or, to update all dependencies:
+  pnpm upgrade
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  # updates the astro dependency:
+  yarn upgrade astro
+  # or, to update all dependencies:
+  yarn upgrade
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+This guide includes major highlights and instructions on how to handle breaking changes when upgrading from v1.x and v2.0-beta releases. See [the changelog](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md) for full release notes.
+
+
+## Astro 2.0 Updates
+
+### Updated: Node Version >=16.12.0
```

Comment:
I'd be more explicit with this title: "Dropping Node v14 Support". With the current title, it's hard to know what the breaking change actually is / what v1 support previously was.